.circleci/config.yml

```diff
@@ -1,17 +1,22 @@
 version: 2.1
+
+orbs:
+  codacy: codacy/base@1.0.0
+
 jobs:
   build: # runs not using Workflows must have a `build` job as entry point
     docker: # run the steps with Docker
       # CircleCI Go images available at: https://hub.docker.com/r/circleci/golang/
       - image: circleci/golang:1.12 #
+    
     # directory where steps are run. Path must conform to the Go Workspace requirements
     working_directory: /go/src/github.com/codacy/helm-ssm
-
+    
     environment: # environment variables for the build itself
       TEST_RESULTS: /tmp/test-results # path to where test results will be saved
 
     steps: # steps that comprise the `build` job
-      - checkout # check out source code to working directory
+      - checkout
```

Comment:
Why do you need to checkout again? Don't you loose the `.version` file this way?

Reply:
You're right! I had to checkout again because the project should be located inside `$GOPATH`, which is different from the directory where workspace is attached.

Reply:
By the way I don't use `.version` file in the `build` job so it is ok 👍